x/profiles/ibc_module.go

```diff
@@ -83,17 +83,17 @@ func (am IBCModule) OnChanOpenInit(
 	channelCap *capabilitytypes.Capability,
 	counterparty channeltypes.Counterparty,
 	version string,
-) error {
+) (string, error) {
 	if err := ValidateProfilesChannelParams(ctx, am.keeper, order, portID, channelID); err != nil {
-		return err
+		return "", err
 	}
 
 	// Claim channel capability passed back by IBC module
 	if err := am.keeper.ClaimCapability(ctx, channelCap, host.ChannelCapabilityPath(portID, channelID)); err != nil {
-		return err
+		return "", err
 	}
 
-	return nil
+	return version, nil
 }
```

Comment:
We didn't define the `ibc-profiles` port version, so this phase we don't check the version.
See:
https://github.com/cosmos/ibc/blob/0cd3e7de753fc44508f4a76269bc2d71a487c059/spec/core/ics-026-routing-module/README.md#chanopeninit